crates/polars-arrow/src/legacy/kernels/rolling/nulls/mod.rs

```diff
@@ -168,38 +168,29 @@ mod test {
 
         let out = rolling_var(arr, 3, 1, false, None, None);
         let out = out.as_any().downcast_ref::<PrimitiveArray<f64>>().unwrap();
-        let out = out
-            .into_iter()
-            .map(|v| v.copied().unwrap())
-            .collect::<Vec<_>>();
+        let out = out.into_iter().map(|v| v.copied()).collect::<Vec<_>>();
 
-        assert_eq!(out, &[0.0, 0.0, 2.0, 12.5]);
+        assert_eq!(out, &[None, None, Some(2.0), Some(12.5)]);
```

Comment:
here the original series is `[1, null, -1, 4]`, and the operation is rolling variance with window of 3 and min_periods 1 and ddof=1

So the windows are:
- [1]
- [1, null]
- [1, null, -1]
- [null, -1, 4]

and so it's correct that the first two need to output `None`, as they only have a single valid value and ddof=1:
```python
In [50]: print(pl.Series([1]).std())
None
```